cours/openstack/openstack-presentation-inside.fr.md

```diff
@@ -151,14 +151,14 @@
 
 - Depuis 2017
```

Comment:
s/2017/2013/ ?

Reply:
No. 2017 is right, according to this report : https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-assets-prod/reports/OpenStack-AnnualReport2017.pdf